CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ Bugfixes:
 Other changes:
 * Hatchling is now used as the build backend. This should have minimal, if any,
   user-facing impact.
+* Special-case support for `_typeshed.Unused` as a `builtins.object` alias.
+  Contributed by [Avasam](https://github.com/Avasam)
```

Comment:
This doesn't describe the user-visible impact very well; how about this?

```suggestion
* Modify Y036 so that `_typeshed.Unused` is allowed as an annotation for
  parameters in `__(a)exit__` methods. Contributed by
  [Avasam](https://github.com/Avasam)
```